### PR TITLE
Redesign dashboard.

### LINKED ISSUE #22: Create notification button in navigation bar of Dashboard
- Remove list of notifications from Dashboard
- Move list of notifications to separate table view
- Open list of notifications via button in navigation bar of Dashboard
- show counter of new notifications on button

### LINKED ISSUE #44: Redesign dashboard for iPad screens
create two column layout?

### PR DESCRIPTION
**Pull request by mathebox**
- Improve iPad design

Fixes #44 
Fixes #22

### CODE REVIEW

File: schulcloud/CoursesViewController.swift [hunk 1]
```diff
@@ -18,6 +18,8 @@ class CoursesViewController: UICollectionViewController, NSFetchedResultsControl
         // self.navigationItem.rightBarButtonItem = self.editButtonItem()
         performFetch()
         updateData()
+
+//        self.collectionView.nu
```
Comment (MasterCarl) [inline schulcloud/CoursesViewController.swift]: Please remove

File: schulcloud/ShortNotificationViewController.swift [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if self.notifications.count == 0 {
+            return 1
+        }
+        return min(self.notifications.count, ShortNotificationViewController.numberOfShownCells)
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cellIdentifier = self.notifications.count == 0 && indexPath.row == 0 ? "emptyListCell" : "notificationCell"
```
Comment (MasterCarl) [inline schulcloud/ShortNotificationViewController.swift]: Why do we check for `indexPath.row == 0` here?

File: schulcloud/ShortNotificationViewController.swift [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        let cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier, for: indexPath)
+
+        if let notificationCell = cell as? NotificationCell {
```
Comment (MasterCarl) [inline schulcloud/ShortNotificationViewController.swift]: I would just use `if self.notifications.count == 0 {} else {}`

**Comment by MasterCarl** [comment]
This also changes views for #12, which I'm working on currently

**Comment by MasterCarl** [comment]
For clarity, I'd add _XCode-specific labels_ to the various nested `StackView`s.

File: schulcloud/ShortNotificationViewController.swift (hunk 3 above)
Comment (mathebox) [reply schulcloud/ShortNotificationViewController.swift]: But then we still have to unwrap the `cell` and check if it is a `NotificationCell`

**Pull request merged**